draft-ietf-acme-acme.md

```diff
@@ -488,11 +490,14 @@ ACME is structured as a REST application with a few types of resources:
   identifier
 * Certificate resources, representing issued certificates
 * A "directory" resource
+* A "new-nonce" resource
 * A "new-registration" resource
 * A "new-application" resource
 * A "revoke-certificate" resource
 * A "key-change" resource
 
+The server MUST provide "directory" and "new-nonce" resources.
+
 For the singular resources above ("directory", "new-registration",
 "new-application", "revoke-certificate", and "key-change") the resource may be
```

Comment:
I think "new-nonce" should be added to this list of singular resources. (Also open to a rewording that means we wouldn't have to keep this in-sync with every new resource added).